tasks/set_facts.yml

```diff
@@ -22,6 +22,14 @@
   when:
     - ansible_os_family == "RedHat"
 
+- name: set_facts | Setting Fedora Pre-Reqs
+  set_fact:
+    kea_dhcp_pre_reqs:
+      - dnf-utils
+  when:
+    - ansible_os_family == "Fedora"
```

Comment:
I assume to should be `ansible_distribution == "Fedora"`, perhaps even complemented by `ansible_os_family == "RedHat"`